### PR TITLE
Add token_expire_user_override link to auth runner docstring

### PR DESCRIPTION
**Pull request by whiteinge**
### What does this PR do?

Cross-link some docs.

### CODE REVIEW

File: salt/runners/auth.py [hunk 1]
```diff
@@ -20,6 +20,9 @@ def mk_token(**load):
     '''
     Create an eauth token using provided credentials
 
+    Non-root users may specify an expiration data if allowed via the
```
Comment (cachedout) [inline salt/runners/auth.py]: `expiration data`? `date`? If so, how is that specified?

**Comment by garethgreenaway** [comment]
Go Go Jenkins!

File: salt/runners/auth.py (hunk 1 above)
Comment (whiteinge) [inline salt/runners/auth.py]: Good catch.

File: salt/runners/auth.py [hunk 2]
```diff
@@ -20,6 +20,10 @@ def mk_token(**load):
     '''
     Create an eauth token using provided credentials
 
+    Non-root users may specify an expiration date (if allowed via the
+    :conf_master:`token_expire_user_override` setting) by passing an additional
+    ``token_expire`` param.
```
Comment (cachedout) [inline salt/runners/auth.py]: It's not clear from reading this what the parameter should *be* though. Below I see an example of it being `94670856` but I have no idea what this means. Could we clarify this a bit, perhaps with an example and explanation?

**Review by cachedout — CHANGES_REQUESTED** [review]

File: salt/runners/auth.py (hunk 2 above)
Comment (whiteinge) [inline salt/runners/auth.py]: Good point, thanks.

**Review by rallytime — APPROVED** [review]

**Pull request merged**